src/test/RwaSpell.t.sol

```diff
@@ -485,28 +485,7 @@ contract DssSpellTest is DSTest, DSMath {
         spell.cast();
     }
 
-    function vote() private {
-        if (chief.hat() != address(spell)) {
-            hevm.store(
-                address(gov),
-                keccak256(abi.encode(address(this), uint256(1))),
-                bytes32(uint256(999999999999 ether))
-            );
-            gov.approve(address(chief), uint256(-1));
-            chief.lock(sub(gov.balanceOf(address(this)), 1 ether));
-
-            assertTrue(!spell.done());
-
-            address[] memory yays = new address[](1);
-            yays[0] = address(spell);
-
-            chief.vote(yays);
-            chief.lift(address(spell));
-        }
-        assertEq(chief.hat(), address(spell));
-    }
-
-    function voteTemp(address _spell) private {
+    function vote(address _spell) private {
         if (chief.hat() != address(_spell)) {
```

Comment:
nit: it could be simplified just using `_spell` as the variable is declared as address